docs/source/devel.rst

```diff
@@ -35,6 +37,30 @@ Tips
 * If a test case exceeds many lines consider creating new keywords.
 * When writing tests, pay attention to the number of spaces.
 
+Naming conventions
+------------------
+
+Robot Framework does not strictly enforce naming conventions for suite names and
```

Comment:
Well, yes, that seems kind of obvious? It doesnt enforce it non-strictly either.  It's just a tool. This has nothing to to with RobotFramework really. I'm not sure way that is a point here.